### ISSUE TITLE
Bar charts

### ISSUE THREAD

**Issue by marcel1334** [mi-erasmusmc/Tantalus#3]
After playing with the tool for a while I miss having a global overview. Its not imediately clear to me how different the vocabs are and where these changers are. Now you have to play with the filters to get this.

Maybe its possible to add some bar charts where you can see what changes there are. You can keep the filters on the left and top and make a tabsheet in the middle with two tabs: table and bar charts. For some columns in the table you can make barcharts (with counts and percentages): Description, Table, Vacabulary, domain. For prevalence and value difference you can make groups (automaticall based on detected values). If you click on one of these bars you can add it to the filtering on top and the charts wil be refreshed using the changed filter. 

In this way you can see immedately see in what area most changes are and zoom in by clicking bar charts. If the filter is set and you are currious what the records are you change the tabsheet in the middel to table and you see the records involved.

I think it would be great to have this extra feature.

**Comment by PRijnbeek** [comment]
Yes I agree. It is indeed nice to add some graphs. I was thinking of adding these in the first place in the Graphical tab on the left (maybe later integrate them with the tables..?). 

Let's brainstorm a bit on the exact contents of these graphs, it should be more than a frequency tab which is shown in the counts in the table on bottom left (if you click on that it will filter already). 

Maybe have a brainstorm session for graphs on the white board and add them as pictures in the this thread?

**Comment by marcel1334** [comment]
Brainstorm is fine. My agenda is free for today. ;-)
